operator/main.py

```diff
@@ -464,22 +466,203 @@ def handle_modified(core_v1_client, obj):
 
     filename = os.path.join(MANIFESTS_DIR, "services.yaml")
     template(filename, namespace=NAMESPACE, volname=volname)
-    execute(KUBECTL_CMD, CREATE_CMD, "-f", filename)
+    lib_execute(KUBECTL_CMD, CREATE_CMD, "-f", filename)
     logging.info(logf("Deployed Service", volname=volname, manifest=filename))
 
+def delete_config_map(core_v1_client, obj):
+    """
+    Volinfo of new hosting Volume is generated and updated to ConfigMap
+    """
+    volname = obj["metadata"]["name"]
+    voltype = obj["spec"]["type"]
+    volume_id = obj["spec"]["volume_id"]
+
+    data = {
+        "namespace": NAMESPACE,
+        "kadalu_version": VERSION,
+        "volname": volname,
+        "volume_id": volume_id,
+        "type": voltype,
+        "bricks": [],
+        "options": obj["spec"].get("options", {})
+    }
+
+    # Add new entry in the existing config map
+    configmap_data = core_v1_client.read_namespaced_config_map(
+        KADALU_CONFIG_MAP, NAMESPACE)
+
+    # For each brick, add brick path and node id
+    bricks = obj["spec"]["storage"]
```

Comment:
Except `name` nothing else will be available in `obj`. You can skip this bricks part